pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -1303,7 +1239,7 @@ def get_iter(x):
             return nc
 
         if search_algorithm is None:
-            search_algorithm = "grid"
+            search_algorithm = "random"  # Defaults to Random
 
         if search_algorithm == "grid":
```

Comment:
We should raise an error if the `search_algorithm` passed is neither `grid` nor `random`.

Reply:
Yes, I will add it in the next round.

Reply:
@Yard1 , I have added this check.